e2e/rbd.go

```diff
@@ -2113,18 +2113,38 @@ var _ = Describe("RBD", func() {
 				if err != nil {
 					e2elog.Failf("failed to delete storageclass : %v", err)
 				}
-				volumeNamePrefix := "test-"
+				scOptions := map[string]string{
+					"volumeNamePrefix": "test-",
+					"encrypted":        "true",
```

Comment:
This will now always run the test `validate the functionality of controller` with an encrypted volume, I guess there should be two tests, one without encryption, and one with encryption (to test all metadata, make it thick-provisioned too!)

Reply:
@nixpanic made the changes PTAL!